include/nn/mem.h

```diff
@@ -7,24 +7,51 @@
 
 #include <nn/os.h>
 #include <nn/types.h>
+#include <nn/util/util_TypedStorage.h>
 
 namespace nn {
+namespace nlibsdk {
+namespace heap {
+class CentralHeap;
+}  // namespace heap
+}  // namespace nlibsdk
+
 namespace mem {
 class StandardAllocator {
 public:
     StandardAllocator();
+    StandardAllocator(void*, u64);
```

Comment:
```suggestion
    StandardAllocator(void* address, u64 size);
```
same for the (void*, u64, bool) overload

Reply:
Fixed!